### ISSUE TITLE
SDL_strlen and SDL_strnlen referenced in docs but undocumented

### ISSUE THREAD

**Issue by DyXel** [libsdl-org/sdlwiki#635]
Some pages such as https://wiki.libsdl.org/SDL3/SDL_strlcat reference it, but lead to a "no such page".

**Comment by icculus** [comment]
Yeah, our biggest documentation coverage gaps right now are these C runtime wrappers. I'll try to take a run at these for 3.2.0, even if just to say something like  "this works exactly like strlen() but doesn't require access to a C runtime."
